crates/node_runtime/src/node_runtime.rs

```diff
@@ -202,6 +217,10 @@ impl NodeRuntime for RealNodeRuntime {
             let mut command = Command::new(node_binary);
             command.env_clear();
             command.env("PATH", env_path);
+            // Set proxy arg, if user's env has `http_proxy`.
```

Comment:
In China, we have to set http_proxy. Otherwise, the npmjs.org can't connect. 

I added this because the below line have `env_clear`, so add this to keep to use `http_proxy` env.